Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -27,38 +27,37 @@ public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListene
         /// <returns>
         /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
         /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
+        public static IDisposable Enable(TelemetryConfiguration configuration = null, IEnumerable<string> excludedDomains = null)
```

Comment:
Nope, `DependencyTrackingTelemetryModule` should be for all scenarios. Customer's code should not differ in .NET Core and Full framework cases. Just do `(new DependencyTrackingTelemetryModule()).Initializer(TelemetryConfiguraiton.Active);` and you have your dependencies collected.

Please change